flybywiresim/a32nx
# docs: A380 active route network

Summary of Changes
Add A380 active routes overview for FBW website. Currently I've only made few of them to see if it's ok like this, once I got confirmation, I will finish the rest of them.

References
I've used FlightRadar24 for route length and average flight time, then Planespotter for fleet info.

Discord username (if different from GitHub): D0m1nuZ @D0m1nu2

How to download the PR for QA
Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

Make sure you are signed in to GitHub
Click on the Checks tab on the PR
On the left side, click on the bottom PR tab
Click on the A32NX download link at the bottom of the page
